README.adoc

```diff
@@ -501,6 +504,22 @@ We want to quickly add reporters for CheckStyle, Jacoco...
 [#db-setup]
 == Database Setup
 
+NOTE: Available since version 1332.xxxxxxxx.xxx
```

Comment:
lines 466 to 468 also talk about H2 and are now wrong, would be nice to fix them also

Reply:
fixed c5e5d3d5e590cc72da0e1a6a1bb7f30667fd1a2f